test/LidoStakedEthRewardsIntegration.t.sol

```diff
@@ -0,0 +1,116 @@
+// SPDX-License-Identifier: AGPL-3.0
+pragma solidity ^0.8.0;
+
+import "forge-std/Test.sol";
+
+import { IAaveOracle }       from "aave-v3-core/contracts/interfaces/IAaveOracle.sol";
+import { IERC20 }            from "aave-v3-core/contracts/dependencies/openzeppelin/contracts/IERC20.sol";
+import { IPoolDataProvider } from "aave-v3-core/contracts/interfaces/IPoolDataProvider.sol";
+
+import { IEACAggregatorProxy }         from "aave-v3-periphery/misc/interfaces/IEACAggregatorProxy.sol";
+import { IEmissionManager }            from "aave-v3-periphery/rewards/interfaces/IEmissionManager.sol";
+import { IRewardsController }          from "aave-v3-periphery/rewards/interfaces/IRewardsController.sol";
+import { PullRewardsTransferStrategy } from "aave-v3-periphery/rewards/transfer-strategies/PullRewardsTransferStrategy.sol";
+import { RewardsDataTypes }            from "aave-v3-periphery/rewards/libraries/RewardsDataTypes.sol";
+
+contract LidoStakedEthRewardsIntegrationTest is Test {
+
+    IAaveOracle        aaveOracle       = IAaveOracle(       0x8105f69D9C41644c6A0803fDA7D03Aa70996cFD9);
+    IPoolDataProvider  poolDataProvider = IPoolDataProvider( 0xFc21d6d146E6086B8359705C8b28512a983db0cb);
+    IEmissionManager   emissionManager  = IEmissionManager(  0xf09e48dd4CA8e76F63a57ADd428bB06fee7932a4);
+    IRewardsController incentives       = IRewardsController(0x4370D3b6C9588E02ce9D22e684387859c7Ff5b34);
+
+    address admin    = 0x3300f198988e4C9C63F75dF86De36421f06af8c4;  // SubDAO Proxy
+    address operator = 0x8076807464DaC94Ac8Aa1f7aF31b58F73bD88A27;  // Operator multi-sig
```

Comment:
Lets add a note that the operator is intended to hold custody of the funds, so will be the rewardVault as well as an emissionAdmin

Reply:
fixed